client/gdc/singlecell.js

```diff
@@ -111,7 +213,7 @@ async function submitSelectedFile(fileId, obj) {
 
 	let result
 	{
-		const body = { genome: gdcGenome, dslabel: gdcDslabel, sample: fileId }
+		const body = { genome: gdcGenome, dslabel: gdcDslabel, sample: fileId, plots: ['UMAP', 'TSNE', 'PCA'] }
```

Comment:
does this code have access to termdbconfig?

Reply:
this submitSelectedFile function (and getFilesAndShowTable) are not used. maybe we delete them?

Reply:
yes please try. in new branch?

Reply:
ok sounds good